test/src/test-reader/controllers/also-controller.ts

```diff
@@ -16,21 +17,25 @@ describe("test-reader/controllers/also-controller", () => {
         return AlsoController.create(eventBus);
     };
 
-    const mkTestObject_ = (
-        { browserId }: { browserId: string } = { browserId: "default-browser-id" },
-    ): ConfigurableTestObject => {
+    const mkTestObject_ = ({
+        browserId = "default-browser-id",
+        parent = null,
+    }: { browserId?: string; parent?: Suite | null } = {}): ConfigurableTestObject => {
         const testObject = new ConfigurableTestObject({
             title: "default-title",
             file: "/default-file",
             id: "default-id",
         });
+
         testObject.browserId = browserId;
+        testObject.parent = parent;
+        testObject.enable = sinon.spy(testObject.enable.bind(testObject));
```

Comment:
Previously, `testObject.enable` was `===` to `testObject.parent.enable`, so i couldn't check "if it was called on parent once". It would refer to call on "testObject.enable" and return "yes, it was called on parent" even if it didn't